### DISCUSSION TITLE
Is there support for auto program

### DISCUSSION THREAD

**Discussion by duskmoon314** [esp-rs/esp-rust-board#51] [Q&A] [answered]
There is no circuit for the auto program in the schema like in the schema of `esp32-c3-devkitm-1`.

I wonder if this board supports the auto program or not.

> BG: I'm new to esp and trying to learn how a board is designed via viewing this project.

**Comment by pedrominatel** [comment, marked as answer]
The embedded USB CDC does the job. In sum, no need to have the reset circuit when flashing over USB CDC from the ESP32-C3.
